README-it.md

```diff
@@ -0,0 +1,82 @@
+# La documentazione di Kubernetes
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+Benvenuto! Questo repository contiene tutte le risorse necessarie per creare il [sito Web e la documentazione di Kubernetes](https://kubernetes.io/). Siamo molto contenti che tu voglia contribuire!
+
+## Contribuire ai documenti
+
+Puoi fare clic sul pulsante **Fork** nell'area in alto a destra dello schermo per creare una copia di questo repository nel tuo account GitHub. Questa copia è chiamata * fork *. Apporta le modifiche desiderate nella tua fork e quando sei pronto a inviarci tali modifiche, vai alla tua fork e crea una nuova richiesta di pull per farcelo sapere.
+
+Una volta creata la richiesta di pull, un revisore di Kubernetes si assumerà la responsabilità di fornire un feedback chiaro e attuabile. Come proprietario della richiesta pull, ** è tua responsabilità modificare la tua richiesta di pull per rispondere al feedback che ti è stato fornito dal revisore di Kubernetes. ** Nota anche che potresti finire con più di un revisore di Kubernetes feedback o si potrebbe finire per ricevere feedback da un revisore di Kubernetes diverso da quello originariamente assegnato per fornire un feedback. Inoltre, in alcuni casi, uno dei revisori potrebbe chiedere una revisione tecnica da un [revisore tecnico di Kubernetes] (https://github.com/kubernetes/website/wiki/Tech-reviewers) quando necessario. I revisori faranno del loro meglio per fornire feedback in modo tempestivo, ma i tempi di risposta possono variare in base alle circostanze.
+
+Per ulteriori informazioni su come contribuire alla documentazione di Kubernetes, vedere:
+
+* [Inizia a contribuire](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Uso dei modelli di pagina](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Guida allo stile della documentazione](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizzazione della documentazione di Kubernetes](https://kubernetes.io/docs/contribute/localization/)
+
+## La documentazione di localizzazione di Kubernetes di `README.md`
+
+### Coreano
```

Comment:
Can you rewrite this section from Korean to Italian so it corresponds with the correct maintainers and slack channels. This is what we do for the German docs as well: #14489

Reply:
sorry, it's not clear this part

Reply:
This section is written in Italian, but is describing the Korean process.
I was aiming at rewriting this to something as:

```text
### Italian

......

Maintainers:

* Michele Berardi ([@micheleberardi](https://github.com/micheleberardi))

Slack channel: .....
```

I think you can keep the translated part but then update the data (names, usernames, link to kubernetes-docs-ko) to explain the Italian structure.